modules/_labs/core/react/lib/theming/README.md

```diff
@@ -160,13 +178,15 @@ const theme: PartialCanvasTheme = {
       main: colors.greenApple400,
     },
   },
+  direction: ContentDirection.RTL
```

Comment:
Both providers are getting RTL. Should this be LTR?

Reply:
yes